dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -32,10 +32,7 @@
 import io.dockstore.common.MuteForSuccessfulTests;
 import io.dockstore.common.SourceControl;
 import io.dockstore.openapi.client.ApiClient;
-import io.dockstore.openapi.client.api.EntriesApi;
-import io.dockstore.openapi.client.api.OrganizationsApi;
-import io.dockstore.openapi.client.api.UsersApi;
-import io.dockstore.openapi.client.api.WorkflowsApi;
+import io.dockstore.openapi.client.api.*;
```

Comment:
Avoid the wildcard 
https://wiki.oicr.on.ca/display/DOC/IDE+Settings and the link to https://github.com/dockstore/dockstore/blob/develop/codestyle.xml#L16-L17 should enforce this. 

https://www.baeldung.com/java-wildcard-imports#disadvantages-of-wildcard-imports for explanation